build-system/tasks/check-types.js

```diff
@@ -188,9 +205,11 @@ async function typeCheck(targetName) {
 
   // If srcGlobs and externGlobs are defined, determine the externs/extraGlobs
   if (srcGlobs.length || externGlobs.length) {
-    opts.eterns = externGlobs.flatMap(globby.sync);
+    opts.externs = externGlobs.flatMap(globby.sync);
```

Comment:
_Eternal joy_ or _external joy_, that is the question 😃

Reply:
🤣